productivity-suite/app/fragments/news/vite.config.ts

```diff
@@ -0,0 +1,42 @@
+import solid from "solid-start/vite";
+import cloudflareWorkers from "solid-start-cloudflare-workers";
+import { defineConfig, PluginOption } from "vite";
+
+function addDefaultFnExportToBundle(): PluginOption {
```

Comment:
this is just a thought, but could we move this function in the `shared` library so that we don't need to re-define it each time? (it can take the regex as a parameter).